### PR TITLE
Get HowTo button to hide

### PR DESCRIPTION
**Pull request by hayleyw7**
# Description drawing

- Hide how-to button on how-to page

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [ ] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -70,9 +70,19 @@ const App = () => {
...
+  const hideHowToBtn = (e) => {
+    const howTo = document.querySelector(".how-to");
+    howTo.classList.add("hidden");
+  }
+
+  const showHowToBtn = (e) => {
+    const howTo = document.querySelector(".how-to");
+    howTo.classList.remove("hidden");
+  }  
+
```
Comment (Matt-Roden) [inline src/components/App/App.js:82]: Nice work, I wonder if there's a different way to do this? Either way it works and I like that :)

**Review by Matt-Roden — APPROVED** [review]

File: src/components/App/App.js (hunk 1 above)
Comment (hayleyw7) [reply src/components/App/App.js:82]: I can look into that today if I get time. This probably isn't the most elegant solution.

Comment (joanafbrito) [reply src/components/App/App.js:82]: Like this one !

**Review by joanafbrito — APPROVED** [review]
Looks nice, thanks !

**Pull request merged**